src/other/Table.js

```diff
@@ -529,27 +567,35 @@
                     }
                 }            
             }
-            context.tableDiv
-                .style("width", finalWidth)
-                .style("height", newTableHeight)
-            ;
+            if (element.classed("childWidget")) {
+                context._parentElement
+                    .style("width", finalWidth)
+                    .style("height", newTableHeight)
+                ;
+                context.tableDiv
+                    .style("overflow", "hidden")
+                ;
+            }
+            context.size({width:parseInt(finalWidth), height: parseInt(newTableHeight)});
```

Comment:
Minor:  Have the calcWidth function return just the width and no "px"...  The reason I prefer this is it means you add the "px" inline when you set the style, thus keeping it "grouped" logically.